.travis.yml

```diff
@@ -18,14 +18,16 @@ branches:
 before_install:
   - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.13.0
   - export PATH="$HOME/.yarn/bin:$PATH"
+  - npm install codecov -g
 
 jobs:
   include:
     - name: "Test"
       script: |
         set -ex
         yarn pre-test
-        yarn test --forbid-only
+        yarn cov-test --forbid-only
+        yarn cov-report-html
```

Comment:
This is not needed, just wasted ... cpu cycles.

Reply:
I took it from our github actions. Should it be removed there as well?

Reply:
I just think that for this particular job, we need only report in `json` format so there is no reason to generate HTML.
OTOH, due to configuration, we already generate HTML report and `coverage/coverage-final.json` in `yarn  cov-test` step.